static/docs/commands-reference/run.md

```diff
@@ -84,6 +84,17 @@ automatically under DVC control. In case of metrics it's pretty usual because
 metric files are small enough to be put into Git or other underlying version
 control system. See also the difference between `-o` and `-O` options.
 
+* `--outs-persist` - the same as `-o` except outputs will not be removed in case
+of [`dvc repro`](https://dvc.org/doc/commands-reference/repro#repro) command. 
+Upon running [`dvc repro`](https://dvc.org/doc/commands-reference/repro#repro), 
+In case of `-o` target file is removed (if exists) before performing command 
```

Comment:
target file is confusing (sounds more like DVC file itself in this context). Why don't we specify that in case of `-o` or `-O` data files or directories produced in the previous run are removed ... or something like this